.github/workflows/ci.yml

```diff
@@ -213,10 +213,24 @@ jobs:
             --build-arg BUILDKIT_INLINE_CACHE=1 \
             --cache-from ghcr.io/dependabot/dependabot-core \
             .
-      - name: Build
-        run: script/build
+      - name: Build dependabot-updater image
+        env:
+          DOCKER_BUILDKIT: 1
+        run: |
+          docker build \
+            -f Dockerfile.updater \
+            -t "dependabot-updater:latest" \
+            --build-arg OMNIBUS_VERSION=latest \
+            --build-arg BUILDKIT_INLINE_CACHE=1 \
+            --cache-from ghcr.io/dependabot/dependabot-updater \
+            .
       - name: Run updater tests
-        run: ./script/ci-test-updater
```

Comment:
I observed that we're no longer running the `./script/ci-test-updater` in CI, but also that script has been modified to take an ecosystem parameter, and similarly with `script/build` above.

I would prefer to keep delegating to scripts in CI, because they make it much easier to try reproduce things locally, but if this is not possible, could/should we remove any of these scripts?

Reply:
I agree, I will take another pass at this after merging and remove a lot of the copy-pasting with script calls. 

The reason I did this here is I wanted to change the script, but not break current CI. So just another copy-paste for the short term...